poseidon2.nimble

```diff
@@ -3,4 +3,4 @@ author = "nim-poseidon2 authors"
 description = "Poseidon2 hash function"
 license = "MIT"
 
-requires "https://github.com/mratsim/constantine#ab6fa6ae1bbbd1b10071a92ec209b381b5d82511"
+requires "https://github.com/mratsim/constantine == 0.1.0"
```

Comment:
i would prefer to pin it to a concrete commit, it's not stable.

Reply:
It is stable now, constantine version 0.1.0 was tagged last week: https://github.com/mratsim/constantine/releases/tag/v0.1.0